scm/github.go

```diff
@@ -335,16 +327,7 @@ func (s *GithubSCM) CreateHook(ctx context.Context, opt *CreateHookOptions) erro
 		},
 		Events: []string{"push", "pull_request", "pull_request_review"},
 	}
-	var err error
-	// prioritize creating an organization hook
-	if opt.Organization != "" {
-		_, _, err = s.client.Organizations.CreateHook(ctx, opt.Organization, hook)
-		if err != nil {
-			return fmt.Errorf("CreateOrgHook: failed to create GitHub hook for org %s: %w", opt.Organization, err)
-		}
-	} else {
-		_, _, err = s.client.Repositories.CreateHook(ctx, opt.Repository.Owner, opt.Repository.Path, hook)
-	}
+	_, _, err := s.client.Organizations.CreateHook(ctx, opt.Organization, hook)
```

Comment:
Should have a comment about this being a course organization hook. Maybe even at the top level (interface) comment.